### PR TITLE
feature: copy channel link menu action

### PR DESCRIPTION
**Pull request by igor-sirotin**
### What does the PR do

Added a channel context menu option to copy link to channel.

I'm not sure why it wasn't added in the first place, but we need to test URL unfurling.
If this button shouldn't exist, or it should be of different design, let me know 🙂 

Screenshot 2023-10-19 at 14 05 22

Screenshot 2023-10-19 at 14 05 28

https://github.com/status-im/status-desktop/assets/25482501/b6644a79-1d12-4d1a-9ba1-971855030ad7

### CODE REVIEW

**Review by jrainville — APPROVED** [review]
This seems like a valid place to get the link.

**Review by alexjba — APPROVED** [review]
LGTM! 👍

**Comment by igor-sirotin** [comment]
I'll wait for @benjthayer approval just in case I didn't match the design

**Comment by jrainville** [comment]
> I'll wait for @benjthayer approval just in case I didn't match the design

As you wish, but IMO it's fine as is. We can just open a new issue/PR once we get the new design

**Comment by alexjba** [comment]
I've been testing the channel url generated and it seems the channel is not recognised by [activateStatusDeepLink](https://github.com/status-im/status-desktop/blob/master/src/app/modules/main/module.nim#L1448) and clicking the channel urls in the message view won't take the user to the channel.

@igor-sirotin Do you know if there's some work in progress in supporting the channel deep links in status-go? Otherwise I guess it's just a bug.

**Comment by igor-sirotin** [comment]
> I've been testing the channel url generated and it seems the channel is not recognised by [activateStatusDeepLink](https://github.com/status-im/status-desktop/blob/master/src/app/modules/main/module.nim#L1448) and clicking the channel urls in the message view won't take the user to the channel.
> 
> @igor-sirotin Do you know if there's some work in progress in supporting the channel deep links in status-go? Otherwise I guess it's just a bug.

I think it was just never implemented. As well as creating the channel link 🙂 
I'll merge this PR and we probably need to open another issue to support clicking channel links 🤔

**Pull request merged**
